kvstore.sh

```diff
@@ -13,30 +13,45 @@ kvstore_usage () {
   echo
   echo "Commands:"
   echo "  ls"
-  echo "    List kv stores (namespaces)"
+  echo "    List kv stores (namespaces)."
+  echo "  lsinfo"
+  echo "    List kv stores (namespaces) and other information."
   echo "  keys <namespace>"
-  echo "    List keys in a namespace"
+  echo "    List keys in a namespace."
   echo "  vals <namespace>"
-  echo "    List values in a namespace"
+  echo "    List values in a namespace."
+  echo "  dump <namespace> [-v]"
+  echo "    List key/values pairs in a namespace. -v prints the"
+  echo "    namespace as a heading."
   echo "  get <namespace> <key>"
-  echo "    Get value of key"
+  echo "    Get value of key."
   echo "  set <namespace> <key> <value>"
-  echo "    Set value of key"
+  echo "    Set value of key."
   echo "  rm <namespace> <key>"
-  echo "    Remove key from store"
+  echo "    Remove key from store."
   echo "  mv <namespace> <from_key> <to_key>"
-  echo "    Move (rename) key.  Fails if the destination key already exists"
+  echo "    Move (rename) key.  Fails if the destination key already exists."
+  echo "  drop <namespace>"
+  echo "    Delete the namespace file."
+  echo "  load"
+  echo "    Used in conjunction with sourcing the file in:"
+  echo "     . \$PATH/TO/kvstore.sh load "
+  echo "    or "
+  echo "     source \$PATH/TO/kvstore.sh load"
+  echo "    Tells the kvstore call at the end of the script not to do anything"
+  echo "    because we are loading all the functions into memory for faster"
+  echo "    execution later on. Add one of the above to your shell profile."
   echo
   echo "Environmental Variables:"
   echo "  KVSTORE_DIR"
-  echo "    Directory where file stores will be kept, defaults to \$HOME/.kvstore"
+  echo "    Directory where file stores will be kept, defaults to '\$HOME/.kvstore'."
   echo
   echo "Shell Initialization"
   echo "  For command completion, add the following to your shell profile:"
   echo "  # File: Shell Profile"
   echo "  \$(kvstore shellinit)"
   echo
-  echo "Version - 2.0"
+  echo "Version - 3.0"
```

Comment:
It seems like you didn't make any breaking changes, so it would be better to make this a minor bump.  I suppose semantic-versioning should also be an explicit standard, though I think of it as a community default.

Reply:
You know, I was torn but you are right - api addition != api breakage so we leave it at 2.1?

Reply:
Fixed.